sdks/python/apache_beam/io/bigquery.py

```diff
@@ -335,6 +336,24 @@ def __init__(self, table=None, dataset=None, project=None, query=None,
     self.validate = validate
     self.coder = coder or RowAsDictJsonCoder()
 
+  def display_data(self):
+    res = {}
+    if self.query is not None:
+      res['query'] = DisplayDataItem(self.query, label='Query')
```

Comment:
Phrased differently: 

* The namespace+key make up the URN for a display data item, useful for UI's to provide richer UI for well-known items (for example: recognize a GCS input path and provide a deep-link)
* The label is the human-readable name of the display item used in UI's. If not explicitly provided, defaults to the key.